pyrefly.toml

```
@@ -0,0 +1,4 @@
+search-path = ["python", "tests/wpt/tests"]
+project-includes = ["python/tidy/**/*.py"]
+project-excludes = ["**/venv/**", "**/.venv/**", "tests/wpt/tests/**", "**/*test*.py", "**/tests/**"]
```

Let's use newlines here to make this a bit more readable:

```suggestion
project-excludes = [
    "**/venv/**", 
    "**/.venv/**", 
    "tests/wpt/tests/**",
    "**/*test*.py",
     "**/tests/**
  ]
```